SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/UserSpanId.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "UserSpanId.generated.h"
+
+USTRUCT(Blueprintable)
+struct FUserSpanId
+{
+	GENERATED_BODY()
+
+	FUserSpanId() {}
+	explicit FUserSpanId(const TArray<uint8>& InData)
+		: Data(InData)
+	{
+	}
+
+	UPROPERTY(BlueprintReadOnly, Category = "UserSpanId")
+	TArray<uint8> Data;
+
+	bool IsValid() const { return Data.Num() == 16; }
```

Comment:
I see the number 16 appearing in so many places, can we define a macro to represent it?